ppr-ui/src/utils/config-helper.ts

```diff
@@ -65,6 +65,10 @@ export async function fetchConfig (): Promise<any> {
   sessionStorage.setItem('KEYCLOAK_CONFIG_PATH', keycloakConfigPath)
   console.info('Set Keycloak Config Path to: ' + keycloakConfigPath)
 
+  const vonApiUrl: string = response.data.VON_API_URL
+  sessionStorage.setItem('VON_API_URL', vonApiUrl)
+  console.log('Set VON API URL to: ' + vonApiUrl)
```

Comment:
doesn't this bark / warn for prod builds?

Reply:
I don't get that when I run it or run the tests/lint is this happening for you? Or are you expecting it to complain for some reason?